api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -57,6 +57,7 @@
 import org.pmiops.workbench.model.AccessBypassRequest;
 import org.pmiops.workbench.model.AccessModule;
 import org.pmiops.workbench.model.Address;
+import org.pmiops.workbench.model.AgreementType;
```

Comment:
Please add these ProfileControllerTests:
* email validation success when agreementType is NULL
* email validation success when agreementType is MASTER
* email validation failure when agreementType is RESTRICTED and the email matches domain
* email validation failure when agreementType is MASTER and the email does not match domain